components/navbar/style.css

```diff
@@ -0,0 +1,80 @@
+header {
+    background-color: #036;
+    border-bottom: 2px solid #fcba19;
+    padding: 0 65px 0 65px;
+    color: #fff;
+    display: flex;
+    height: 65px;
+    top: 0px;
+    position: fixed;
+    width: 100%;
+  }
+  
+  header h1 {
+    font-family: 'PT Sans';
+    font-weight: normal;  /* 400 */
+    margin: 5px 5px 0 18px;
+    visibility: hidden;
+  }
+  
+  header .banner {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 0 10px 0 0;
+    border-style: dotted;
+    border-width: 1px;
+    border-color: lightgrey;
+  }
+  
+  header .other {
+    display: flex;
+    flex-grow: 1;
+    border-style: dotted;
+    border-width: 1px;
+    border-color: lightgrey;
+  }
+
+  header .nav-btn {
+    position: absolute;
+    top: 15px;
+    right: 10px;
+    z-index: 100;
+    width: auto;
+    margin: 0;
+    /* padding: 5px 8px 4px;
+    /* border-color: #fcba19;
+    background: #036; */
+  }
+
+  header a [class^="fas fa-"] {
```

Comment:
@patricksimonian Not sure if this was needed but if I understand it correctly it'll style any FA icons in the `header`. Overkill?

Reply:
i think it  may be better left off to the implementers since they may choose to use a different icon library like bootstrap icons or even FontAwesome's svg icon library

Reply:
Do you think I should be doing this with the SVG icon set?